### ISSUE TITLE
sumtable doesn't include group names in table with out = "kable"

### ISSUE THREAD

**Issue by bberger94** [NickCH-K/vtable#21]
If you run this code Group 1 and Group 2 won't have their own columns in the resulting kable, even though it does work with other output formats.

```
library(vtable)

n <- 100
data <- data.frame(
  g = ifelse(rbinom(n, 1, .5) == 1, "Group 1", "Group 2"),
  y = rnorm(n)
)

sumtable(data, group = "g", out = "kable", group.test = T) 
```

**Comment by NickCH-K** [comment]
Hmm, that's a side effect of `kable` not being able to handle multi-column cells. But it should have the same fix as the `data.frame` return does.

I'll work on this. For now you can get around it by instead doing `knitr::kable(sumtable(data, group = "g", out = "return", group.test = T))` or using an `out` option that respects multi-column cells.

**Comment by bberger94** [comment]
Thanks for looking into this and generally for this wonderful package!

**Comment by NickCH-K** [comment]
Fixed with 85fbde5e4aaa6244cfdc189e7add30e9f47cec19

It was an issue where `out = 'kable'` wasn't being processed properly if run outside of `knitr`.
